### PR TITLE
Add global translations.

### PR DESCRIPTION
**Pull request by Furgas**
Might be treated as conceptual work for issue #5 or merged:) I went with one translation file per holiday, beacuse it's much easier to track what's missing.

### CODE REVIEW

**Comment by stelgenhof** [comment]
Wow! Looking very good. I am reading through it to understand what you did.

**Comment by stelgenhof** [comment]
Can you also add unit tests? Also some functions are missing phpdoc :)

**Comment by Furgas** [comment]
I will add unit tests soon.
I left out PHPDoc on methods implementing documented interfaces. What's your preference on this?

**Comment by stelgenhof** [comment]
I personally like every function to be documented :)

File: src/Yasumi/Holiday.php [hunk 1]
```diff
@@ -77,7 +78,7 @@ class Holiday extends DateTime implements JsonSerializable
      *
      * @param string   $shortName            The short name (internal name) of this holiday
      * @param array    $names                An array containing the name/description of this holiday in various
-     *                                       languages
+     *                                       languages. Ovverrides global translations
```
Comment (stelgenhof) [inline src/Yasumi/Holiday.php]: Typo: should be "Overrides"

**Comment by stelgenhof** [comment]
Do you want me to merge your PR or do you want to update it first?

**Comment by Furgas** [comment]
I will update in a moment.

**Comment by Furgas** [comment]
I think that it's ready to merge.

**Comment by stelgenhof** [comment]
Great! Let me have a look.

**Pull request merged**